hierarchy-tool/tool/src/webapp/WEB-INF/velocity/show.vm

```diff
@@ -3,91 +3,217 @@
 
 #chef_portletContainer()
 #chef_portletContent()
+<style>
+    table, th, td {
+        border-collapse: collapse;
+    }
+    th, td {
+        padding: 5px;
+        text-align: left;
+        vertical-align:bottom;
+    }
+    .label_h3 {
+        color: #555;
+        padding: 0;
+        font-weight: bold;
+        font-family: 'PT Sans', 'Helvetica Neue', Arial, sans-serif;
+        background-color: transparent !important;
+    }
+
+</style>
+<table class="lines nolines" width="50%">
+    <tr>
+        <td width="2em">
+            <form method="post" action="${rootUrl}/site/change">
+                <input type="submit" value="#springMessage("button.replace.site")" #if(!$canReplace)disabled="true"#end/>
+            </form>
+        </td>
+        <td width="2em">
+            <form method="GET" action="${rootUrl}/delete">
+                <input type="submit" value="#springMessage("button.remove.site")"/>
+            </form>
+        </td>
+        <td>
+            #if($canMove)
+                <form method="POST" action="${rootUrl}/cut">
+                    <input type="submit" value="#springMessage("button.move.site")"/>
+                </form>
+            #end
+        </td>
+    </tr>
+</table>
+<table class="lines nolines">
+  <tr>
+
+        <td>
+            #if ($current)
+            <h3>Site Title: </h3>
+
+        </td>
+        <td padding="5px" valign="bottom">
+                <label>${current.siteTitle}</label>
+        </td>
+  </tr>
+  <tr>
+        <td padding="5px">
+            #if ($current.nodeUrl)
+                <h3>#springMessage("show.site.url"):</h3>
+         </td>
+         <td padding="5px" valign="bottom">
+                <a href="$current.nodeUrl" target="_top">$current.nodeUrl</a>
+            #end
+        </td>
+    </tr>
+    <tr>
+        <td padding="5px">
+            #if ($current.siteShortDescription)
+                <h3>#springMessage("show.site.desc"):</h3>
+         </td><td>
+                <label>$current.siteShortDescription</label>
+            #end
+        </td>
+    </tr>
+    <tr>
+        <td padding="5px">
+                #if ($other.size() > 0)
+                <h3>#springMessage("show.other"):</h3>
+        </td>
+        <td>
+                <dd>
+                    <ul>
+                        #foreach ($node in $other)
+                            <li><a href="$node.nodeUrl" target="_top">$node.nodeUrl</a></li>
+                        #end
+                    </ul>
+                </dd>
+            #end
+            #else
```

Comment:
From a style point of view this else is really hard to read as there's no clue from indentation or other means that this matches the \#if on line 49.